code/obj/item/storage/clothing.dm

```diff
@@ -83,6 +83,7 @@
 	/obj/item/device/radio/headset/civilian,\
 	/obj/item/device/pda2/botanist,\
 	/obj/item/clothing/gloves/black,\
+	/obj/item/clothing/glasses/phyto,\
```

Comment:
This should be in the locker itself, not the clothing box, see research's clothing boxes.